steps/appdome-build-2secure-android/1.0.1/step.yml

```diff
@@ -0,0 +1,52 @@
+title: Appdome Build
```

Comment:
The title has the same name on both platforms which will be hard to distinguish on the Workflow Editor search view if someone searches the all steps instead of the platform specific ones. 

So in your case I would add `iOS` and `Android` respectively into the title to make this clear.

Reply:
fix done.